runtime/src/assets/stylesheets/shared/main.less

```diff
@@ -0,0 +1,76 @@
+@import (reference) '_custom-variables.less';
+@import (reference) '_custom-mixins.less';
+body {
+  font-family: Open Sans;
+  font-size: 14px;
+}
+//colors
+.color-grey { color: @color-pf-black-800 !important; }
+// custom classes
+.containerPadd {
+  padding-top: 96px;
+  padding-bottom: 66px;
+}
+.containerPadd-nologin { padding-bottom: 0; }
+.truncate { .truncate; }
+.left-truncate {
+  overflow: hidden;
+  text-overflow: ellipsis;
+  text-overflow: clip;
+  white-space: nowrap;
+  background-color: @color-pf-white;
+  &:extend(.posRel);
+  &:before {
+    position: absolute;
+    z-index: 1;
+    left: -12px;
+    padding-left: 12px;
+    margin-left: 6px;
+    background-color: inherit;
+    content: '\02026';
+  }
+  span {
+    position: relative;
+    display: inline-block;
+    float: right;
+    min-width: 100%;
+    overflow: visible;
+    text-indent: 6px;
+    background-color: inherit;
+    &:before {
+      position: absolute;
+      z-index: 200;
+      left: -8px;
+      width: 12px;
+      height: 24px;
+      background-color: inherit;
+      content: '';
+      &:extend(.dib);
+    }
+  }
+}
+.truncate-ellipsis {
+  display: table;
+  table-layout: fixed;
+  white-space: nowrap;
+}
+.dn { display: none; }
+.dib { display: inline-block; }
+.db { display: block; }
+.textCenter { text-align: @textCenter; }
```

Comment:
this is confusing - declaring a variable that then has a mixin inside of it? can be simplified as `.textCenter { text-align: center; }`